qiskit/algorithms/amplitude_estimators/iae.py

```diff
@@ -52,7 +57,8 @@ def __init__(
         alpha: float,
         confint_method: str = "beta",
         min_ratio: float = 2,
-        quantum_instance: Optional[Union[QuantumInstance, Backend]] = None,
+        quantum_instance: None | QuantumInstance | Backend = None,
```

Comment:
```suggestion
        quantum_instance: QuantumInstance | Backend | None = None,
```
Let's perhaps apply this change everywhere as discussed in the channel.